VideoLocker/src/main/java/org/edx/mobile/view/DiscussionAddPostFragment.java

```diff
@@ -264,4 +265,17 @@ public void onSuccess(CourseTopics courseTopics) {
         getTopicListTask.execute();
     }
 
+    @Override
+    public void onResume() {
+        super.onResume();
+
+        if (getResources().getConfiguration().orientation == Configuration.ORIENTATION_LANDSCAPE) {
```

Comment:
Personally, I think it's fine as is.

Maybe in the future we can move this logic to a single method/class, but for now, it feels a bit like over engineering.